### ISSUE TITLE
Colspan in tables

### ISSUE THREAD

**Issue by SophieBree** [tgrosinger/advanced-tables-obsidian#55]
Hi, firstly I'd like to say I'm really liking the Advanced Tables plugin and I find it very useful for making basic tables. I've been trying to implement some more advanced formatting and I'd like to know if this is possible, or I'm just making a mistake:

On the website [Markdown Here](https://markdown-here.com/livedemo.html), I inserted this Markdown and it outputted the desired result, which is the ability to span a cell over multiple cells in a table.

![image](https://user-images.githubusercontent.com/44855588/104839898-52e8ae80-5918-11eb-8929-935f7b73af27.png)

But when I tried copying this directly into Obsidian, and formatting with Advanced Tables, it outputted as this:

![image](https://user-images.githubusercontent.com/44855588/104839972-b5da4580-5918-11eb-8b22-cd4ba5a638ce.png)

As you can see, there are empty cells hanging over the side on the second row. Is it possible to make the table look like it does in the first image, in Obsidian?

To summarise, I'm looking for the ability to use Advanced Tables to make tables in Obsidian that let me use colspan and rowspan without creating empty cells that I can't get rid of.

Thank you

**Comment by tgrosinger** [comment]
Hi @SophieBree, 
Some flavors of Markdown allow column spans, but it is not part of the core markdown spec. The Advanced Tables plugin does not have any effect on the preview mode of tables only edit mode. Because of this it does not have the ability to add support for column spans to Obsidian.

You might try asking the Obsidian developers in the forum to add support for column spans.

Sorry I can't give you a more positive response!
